expertise/create_dataset.py

```diff
@@ -85,17 +87,59 @@ def deduplicate_publications(self, publications):
                 deduplicated.append(pub)
 
         return deduplicated
+    
+    def get_pub_weight(self, pub=None, venueid=None, weight_specification=None):
```

Comment:
no need to pass a venueid as param, you can extract ir from the pub note, right?

Reply:
Changed in recent commit